atlasdb-cli/build.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'org.inferred.processors'
 
 dependencies {
     compile project(':atlasdb-cassandra')
-    compile project(path: ':atlasdb-dagger', configuration: 'shadow')
+    compile project(':atlasdb-dagger')
```

Comment:
Why don't we just add a `compileOnly` dependency on dagger with a message about why it is needed?